### DISCUSSION TITLE
Request for Domain Addition

### DISCUSSION THREAD

**Discussion by suzuki-rintaro** [RosettaCommons/rosetta#249] [ROSIE] [answered]
Hello,

I encountered the message "Your GitHub account does not have a verified academic email address!" when trying to log in to ROSIE with my GitHub account.

I am affiliated with Japan’s National Agriculture and Food Research Organization, and my email address is rsuzuki@affrc.go.jp.

Could you please add this domain to the list of approved academic domains?

**Comment by lyskov** [comment, marked as answer]
— Thank you for letting us know! This should be fixed now, if you still have issues logging-in please feel free to re-open this issue.
